### DISCUSSION TITLE
How to only block IPv4 results while keeping other DNS query results

### DISCUSSION THREAD

**Discussion by Bitcion** [AdguardTeam/AdGuardHome#6494] [Q&A]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### The problem

I am using the rule "||fast.com^$dnstype=~A|~CNAME" to block IPv6 results. Then what rule should be used to block IPv4 results? I tried "$dnstype=~aaaa|~CNAME" but it does not seem to take effect.

### Proposed solution

only block IPv4

**Comment by a904055262** [comment]
$dnstype=A,dnsrewrite=NOERROR

**Comment by a904055262** [comment]
block ipv6
$dnstype=AAAA,dnsrewrite=NOERROR
